sockeye/inference.py

```diff
@@ -869,11 +948,20 @@ def translate(self, trans_inputs: List[TranslatorInput], fill_up_batches: bool =
         chunks_by_input_idx = itertools.groupby(translated_chunks, key=lambda translation: translation.input_idx)
         for trans_input, (input_idx, translations_for_input_idx) in zip(trans_inputs, chunks_by_input_idx):
             translations_for_input_idx = list(translations_for_input_idx)  # type: ignore
+            num_target_prefix_tokens = trans_input.num_target_prefix_tokens()
```

Comment:
[nit] If you make `TranslatorInput.num_target_prefix_tokens` a `@property` you can save this line and just use `trans_input.num_target_prefix_tokens` directly below.

Reply:
makes sense, thank you!